i18n/languages/woocommerce-mercadopago.pot

```diff
@@ -812,7 +812,7 @@ msgid "Production"
 msgstr ""
 
 #: includes/payments/WC_WooMercadoPago_PaymentAbstract.php:627
-msgid "Choose YES only when you are ready to sell. Change to NO to activate the Tests mode."
+msgid "Choose “Yes”only when you’re ready to sell. Switch to “No” to activate Testing mode."
```

Comment:
msgid "Choose “Yes” only when you’re ready to sell. Switch to “No” to activate Testing mode."
(espaço depois do "Yes")